extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html

```diff
@@ -29,10 +32,20 @@ <h5>REST Resources</h5>
  {/if}
  {/for}
  <h5>Static Resources</h5>
- {#for staticResources in info:staticResourcesInfo}
+ {#for resource in info:staticResourceInfo.resourceMap.get("/").children}
  <div class="row">
   <div class="col">
-   <a href="{staticResources}">{staticResources}</a>
+   {#if resource.isFolder}
+   <i class="far fa-folder-open" style="margin-right:5px; margin-top:5px"></i>
+   <a class="static" href="http://localhost:8080/q/dev/io.quarkus.quarkus-resteasy-reactive/view?path=/{resource.lastName}">
+    {resource.lastName}
+   </a>
+   {#else}
+   <i class="fas fa-file-alt" style="margin-right:5px; margin-top:5px"></i>
+   <a class="static" href="http://localhost:8080/{resource.lastName}">
```

Comment:
Here use an absolute path, but without the host: `/{resource.lastName}`

Reply:
If I am only writing `<a class="static" href="{resource.lastName}">` then it is opening up this link : http://localhost:8080/q/dev/io.quarkus.quarkus-resteasy-reactive/index.html  with the text Resource Not Found.

Reply:
Done